### PR TITLE
Meeting class created

### PR DESCRIPTION
**Pull request by abdelrahmenAyman**

### CODE REVIEW

File: app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Created by abdelrahmen on 11/03/17.
+ */
+
+public class Meeting {
+
+    private Date start_date;
+    private Date end_Date;
+    private String goals;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java]: change the goals attribute to JSONObject

File: app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    private String duration;
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java]: duration is an integer in minutes

File: app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+    public ArrayList<Attendee> getAttendees() {
+        return attendees;
+    }
+
+    public User getSupervisor() {
+        return supervisor;
+    }
+
+    private static class Attendee{
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java]: change the identifier of the class to public

File: app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
+package org.ieeeguc.ieeeguc.models;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by abdelrahmen on 11/03/17.
```
Comment (KEMDAK) [inline app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java]: remove this comment

**Review by KEMDAK — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java (hunk 4 above)
Comment (abdelrahmenAyman) [reply app/src/main/java/org/ieeeguc/ieeeguc/models/Meeting.java]: auto generated :D

**Review by KEMDAK — APPROVED** [review]

**Pull request merged**